### PR TITLE
added pt_br

### PR DESCRIPTION
**Pull request by FITFC**

### CODE REVIEW

File: src/main/resources/assets/merequester/lang/pt_br.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "itemGroup.merequester": "ME Solicitante",
+  
+    "item.merequester.requester_terminal": "ME Terminal do Solicitante",
+    "block.merequester.requester": "ME Solicitante",
+  
+    "gui.merequester.requester_terminal": "ME Terminal do Solicitante",
+    "gui.merequester.no_requesters": "Nenhum solicitante de ME encontrado!",
```
Comment (rlnt) [inline src/main/resources/assets/merequester/lang/pt_br.json:8]: Have you made sure this fits on the screen? It's the text that is displayed inside the terminal when no ME Requesters are found on the network.

File: src/main/resources/assets/merequester/lang/pt_br.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+    "itemGroup.merequester": "ME Solicitante",
+  
+    "item.merequester.requester_terminal": "ME Terminal do Solicitante",
+    "block.merequester.requester": "ME Solicitante",
+  
+    "gui.merequester.requester_terminal": "ME Terminal do Solicitante",
```
Comment (rlnt) [inline src/main/resources/assets/merequester/lang/pt_br.json:7]: Have you made sure this fits on the screen? It's the label shown next to the search text box at the top of the terminal.

**Comment by FITFC** [comment]
Yes it fits... however my resolution is high so I'm not a good example to check this.

**Comment by rlnt** [comment]
Okay, I will try it out myself when I find the time, to ensure. Thank you for your contribution. :)

**Pull request merged**